### ISSUE TITLE
CreateClaimableBalanceResult balanceID is always null

### ISSUE THREAD

**Issue by jopmiddelkamp** [Soneso/stellar_flutter_sdk#40]
The code to reproduce the issue.

```dart
final sdk = StellarSDK.TESTNET;
final network = Network.TESTNET;

final sourceKeyPair = KeyPair.fromSecretSeed(
  "SAW74SJ4ERDAKW4GSL63ASQAN7EVFCI4A44JMRKPPGAP6ROGPBL6SN3J",
  // GAS4N4UW4CU24AIVQIIGDW6ENUYZUVZ7Z3MH5GVYGQHVKXLJ2GBDP6RQ
);

await FriendBot.fundTestAccount(sourceKeyPair.accountId);

final createClaimableBalanceBuilder = CreateClaimableBalanceOperationBuilder(
  [
    Claimant(
      sourceKeyPair.accountId,
      Claimant.predicateNot(
        Claimant.predicateBeforeRelativeTime(60),
      ),
    )
  ],
  Asset.NATIVE,
  "100",
);

final sourceAccount = await sdk.accounts.account(
  sourceKeyPair.accountId,
);

final transactionBuilder = TransactionBuilder(sourceAccount)
  ..addOperation(createClaimableBalanceBuilder.build());

final transaction = transactionBuilder.build()..sign(sourceKeyPair, network);

final result = await sdk.submitTransaction(transaction);
```

**Comment by christian-rogobete** [comment]
Hello @jopmiddelkamp,
thank you for adding this issue. I will have a look to it at the beginning of next week.
Best regards

**Comment by christian-rogobete** [comment]
fixed in [release 1.3.1](https://github.com/Soneso/stellar_flutter_sdk/releases/tag/1.3.1)
